application.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+import click
+
+
+@click.command()
+@click.argument('path', default='')
+@click.argument('word', default='')
+def word_search(path, word):
+    """takes the file path and the word from the user and search for existence of the word inside the file from the
+    path """
+    validation(path, word)
+
+    if path.endswith(".txt") | path.endswith(".json"):
+        try:
+            with open(os.path.join(os.path.dirname(__file__), path), 'r') as input_file:
+                content = input_file.read()
+            x = content.__contains__(word)
+            if x:
+                sys.exit("searchWord Found")
```

Comment:
I missed one point in the POC. If you could return the number of occurrences of the word instead of  'search word found' that would be better.

Reply:
implemented this.